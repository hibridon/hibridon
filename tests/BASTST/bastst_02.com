inp=bastst_02.inp
out=bastst_02.out
run
save=save_02.inp