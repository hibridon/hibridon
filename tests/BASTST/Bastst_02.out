      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =  -1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   0
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 2-SIGMA system parameters:
 NTERM  =   1        NRMAX  =  10        NPAR   =   2        ISYM   =   1
 IGU    =   1        ISA    =   1
 BROT   =  2.000     GSR    = 1.0000E-02 DROT   = 5.0000E-05 HROT   = 1.0000E-02
 LAMMIN:    1
 LAMMAX:   10
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT:  4; JOUT:  0   2   4   6
 INDOUT:    0
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-01-17 at 11:22
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 4a68289e6183a36dae41c0fb214e430c24fc6e3f
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Jan 17 11:27:28 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISTROPIC TERMS IN POTENTIAL = 10

 **  CC 2SIGMA ** RMU=  16.4700  BROT=  2.000  G-SR= 0.100E-01  +/-= 1
     E=5000.00  JTOT= 20.5   JLPAR=-1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   J  EPS   N    L   EINT(CM-1)
   1  0.5   1   0    21     0.000
   2  0.5  -1   1    20     4.070
   3  1.5   1   1    20     4.085
   4  1.5   1   1    22     4.085
   5  1.5  -1   2    19    14.143
   6  1.5  -1   2    21    14.143
   7  2.5   1   2    19    14.168
   8  2.5   1   2    21    14.168
   9  2.5   1   2    23    14.168
  10  2.5  -1   3    18    41.253
  11  2.5  -1   3    20    41.253
  12  2.5  -1   3    22    41.253
  13  3.5   1   3    18    41.288
  14  3.5   1   3    20    41.288
  15  3.5   1   3    22    41.288
  16  3.5   1   3    24    41.288
  17  3.5  -1   4    17   119.955
  18  3.5  -1   4    19   119.955
  19  3.5  -1   4    21   119.955
  20  3.5  -1   4    23   119.955
  21  4.5   1   4    17   120.000
  22  4.5   1   4    19   120.000
  23  4.5   1   4    21   120.000
  24  4.5   1   4    23   120.000
  25  4.5   1   4    25   120.000
  26  4.5  -1   5    16   329.925
  27  4.5  -1   5    18   329.925
  28  4.5  -1   5    20   329.925
  29  4.5  -1   5    22   329.925
  30  4.5  -1   5    24   329.925
  31  5.5   1   5    16   329.980
  32  5.5   1   5    18   329.980
  33  5.5   1   5    20   329.980
  34  5.5   1   5    22   329.980
  35  5.5   1   5    24   329.980
  36  5.5   1   5    26   329.980
  37  5.5  -1   6    15   824.757
  38  5.5  -1   6    17   824.757
  39  5.5  -1   6    19   824.757
  40  5.5  -1   6    21   824.757
  41  5.5  -1   6    23   824.757
  42  5.5  -1   6    25   824.757
  43  6.5   1   6    15   824.822
  44  6.5   1   6    17   824.822
  45  6.5   1   6    19   824.822
  46  6.5   1   6    21   824.822
  47  6.5   1   6    23   824.822
  48  6.5   1   6    25   824.822
  49  6.5   1   6    27   824.822
  50  6.5  -1   7    14  1867.963
  51  6.5  -1   7    16  1867.963
  52  6.5  -1   7    18  1867.963
  53  6.5  -1   7    20  1867.963
  54  6.5  -1   7    22  1867.963
  55  6.5  -1   7    24  1867.963
  56  6.5  -1   7    26  1867.963
  57  7.5   1   7    14  1868.038
  58  7.5   1   7    16  1868.038
  59  7.5   1   7    18  1868.038
  60  7.5   1   7    20  1868.038
  61  7.5   1   7    22  1868.038
  62  7.5   1   7    24  1868.038
  63  7.5   1   7    26  1868.038
  64  7.5   1   7    28  1868.038
  65  7.5  -1   8    13  3876.176
  66  7.5  -1   8    15  3876.176
  67  7.5  -1   8    17  3876.176
  68  7.5  -1   8    19  3876.176
  69  7.5  -1   8    21  3876.176
  70  7.5  -1   8    23  3876.176
  71  7.5  -1   8    25  3876.176
  72  7.5  -1   8    27  3876.176
  73  8.5   1   8    13  3876.261
  74  8.5   1   8    15  3876.261
  75  8.5   1   8    17  3876.261
  76  8.5   1   8    19  3876.261
  77  8.5   1   8    21  3876.261
  78  8.5   1   8    23  3876.261
  79  8.5   1   8    25  3876.261
  80  8.5   1   8    27  3876.261
  81  8.5   1   8    29  3876.261
  82  8.5  -1   9    12  7469.545
  83  8.5  -1   9    14  7469.545
  84  8.5  -1   9    16  7469.545
  85  8.5  -1   9    18  7469.545
  86  8.5  -1   9    20  7469.545
  87  8.5  -1   9    22  7469.545
  88  8.5  -1   9    24  7469.545
  89  8.5  -1   9    26  7469.545
  90  8.5  -1   9    28  7469.545
  91  9.5   1   9    12  7469.640
  92  9.5   1   9    14  7469.640
  93  9.5   1   9    16  7469.640
  94  9.5   1   9    18  7469.640
  95  9.5   1   9    20  7469.640
  96  9.5   1   9    22  7469.640
  97  9.5   1   9    24  7469.640
  98  9.5   1   9    26  7469.640
  99  9.5   1   9    28  7469.640
 100  9.5   1   9    30  7469.640
 101  9.5  -1  10    11 13529.340
 102  9.5  -1  10    13 13529.340
 103  9.5  -1  10    15 13529.340
 104  9.5  -1  10    17 13529.340
 105  9.5  -1  10    19 13529.340
 106  9.5  -1  10    21 13529.340
 107  9.5  -1  10    23 13529.340
 108  9.5  -1  10    25 13529.340
 109  9.5  -1  10    27 13529.340
 110  9.5  -1  10    29 13529.340
 111 10.5   1  10    11 13529.445
 112 10.5   1  10    13 13529.445
 113 10.5   1  10    15 13529.445
 114 10.5   1  10    17 13529.445
 115 10.5   1  10    19 13529.445
 116 10.5   1  10    21 13529.445
 117 10.5   1  10    23 13529.445
 118 10.5   1  10    25 13529.445
 119 10.5   1  10    27 13529.445
 120 10.5   1  10    29 13529.445
 121 10.5   1  10    31 13529.445
ILAM=  1 LAMNUM(ILAM) =    300
ILAM=  2 LAMNUM(ILAM) =    743
ILAM=  3 LAMNUM(ILAM) =   1101
ILAM=  4 LAMNUM(ILAM) =   1631
ILAM=  5 LAMNUM(ILAM) =   1971
ILAM=  6 LAMNUM(ILAM) =   2421
ILAM=  7 LAMNUM(ILAM) =   2614
ILAM=  8 LAMNUM(ILAM) =   2885
ILAM=  9 LAMNUM(ILAM) =   2870
ILAM= 10 LAMNUM(ILAM) =   2929
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS 19465
ilam =   1 :        300/     14641 non zero elements (sparsity:   2.05%)
ilam =   2 :        743/     14641 non zero elements (sparsity:   5.07%)
ilam =   3 :       1101/     14641 non zero elements (sparsity:   7.52%)
ilam =   4 :       1631/     14641 non zero elements (sparsity:  11.14%)
ilam =   5 :       1971/     14641 non zero elements (sparsity:  13.46%)
ilam =   6 :       2421/     14641 non zero elements (sparsity:  16.54%)
ilam =   7 :       2614/     14641 non zero elements (sparsity:  17.85%)
ilam =   8 :       2885/     14641 non zero elements (sparsity:  19.70%)
ilam =   9 :       2870/     14641 non zero elements (sparsity:  19.60%)
ilam =  10 :       2929/     14641 non zero elements (sparsity:  20.01%)
total :      19465/    146410 non zero elements (sparsity : 13.29%)
storage efficiency :  77.57% (         233580 used bytes /          301104 allocated bytes)
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    6.2 %
      CURRENT DATE:  Jan 17 11:27:28 2023
 ===============================================================================
